resources/views/movies/edit-form.blade.php

```diff
@@ -0,0 +1,35 @@
+@extends('layout')
+
+@section('title', 'Edit Film')
+
+@section('content')
+    <form action="{{ route('movies.edit.card', ['id' => $movie->id]) }}" method="post">
+        @csrf
+
+        <div class="mb-3">
+            <label for="title" class="form-label">{{ __('validation.attributes.title') }}</label>
+            <input type="text" value="{{ old($movie->title) }}" class="form-control  @error('title') is-invalid @enderror" aria-describedby="emailHelp" name="title">
```

Comment:
это чтобы в форму подставлялись последние отправленные данные
а если ничего не отправляли еще, то подставлять данные из модели
```suggestion
            <input type="text" value="{{ old('title', $movie->title) }}" class="form-control  @error('title') is-invalid @enderror" aria-describedby="emailHelp" name="title">
```

для полеq year of issue и descrtiption тоже надо поправить